Project-Hotpot/Project-Hotpot/ViewControllers/HomeViewController.swift

```diff
@@ -62,12 +62,16 @@ class HomeViewController: UIViewController {
                     
                 })
                 //TODO: CONSIDER STRUCTURE OF CODE, you repeate these 3 methods in both clauses of if statement
+                //add to history
                 self.saveCurrentSong()
+                //change the song
                 self.resetSong()
+                //change the card
                 self.resetCard()
```

Comment:
Minor code smell here: having comments that are almost as long as the method names, but use different words, make it seem like it would be better to just have different method names.